File: Labs/Lab1/Lab1.java
import java.util.ArrayList;
import java.util.Scanner;

public class Lab1 {
	public static void main(String[] args) {
		Scanner in = new Scanner(System.in);
		ArrayList<Integer> nums = new ArrayList<Integer>();
		System.out.println("Target Number?: ");
		int target = in.nextInt();
		System.out.println("How many integers in array?: ");
		int amt = in.nextInt();
		for (int i = 0; i < amt; i ++) {
			nums.add(i);
		}
		for (int i = 0; i < nums.size(); i++) {
			for (int j = i + 1; j < nums.size(); j++) {
				if (nums.get(i) + nums.get(j) == target) {
					System.out.println(nums.get(i) + " + " + nums.get(j) + " = " + target);
					return;
				}
			}
		}
		System.out.println("Target is too big for these integers!");
	}
}

File: Labs/Lab2/Lab1.py
import itertools

# No need to create a class in Python for this simple script
def main():
    target = int(input("Target Number?: "))
    amt = int(input("How many integers in array?: "))
    nums = list(range(amt))
    
    for i, j in itertools.combinations(nums, 2):
        if i + j == target:
            print(f"{i} + {j} = {target}")
            return
    
    print("Target is too big for these integers!")

if __name__ == "__main__":
    main()

